Makefile

```diff
@@ -39,9 +41,19 @@ test:
 .PHONY: clean
 clean:
 	- rm -f tc-redirect-tap
-
-deps:
-	echo
-
-lint:
-	echo
+	- rm -rf $(BINPATH)
+	- rm -rf .*.stamp
+
+.PHONY: deps
+deps: .lint.stamp
+
+.lint.stamp:
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(BINPATH) v1.46.2
+	$(BINPATH)/golangci-lint --version
+	GOBIN=$(BINPATH) GO111MODULE=off go get -u github.com/vbatts/git-validation
+	@touch $@
+
+.PHONY: lint
+lint: .lint.stamp
+	$(BINPATH)/git-validation -run DCO,short-subject -range HEAD~8..HEAD
```

Comment:
Can't do the full twenty for two reasons:

1. Initial commit from GitHub automation fails DCO check.
2. A previous commit does not have DCO and was ugly to fix. At least within this PR.